src/components/PopupWithForm.js

```diff
@@ -29,7 +29,8 @@ export default class PopupWithForm extends Popup {
     return inputsObject;
   }
 
```

Comment:
```js

  open() {
    super.open();
    this._setEventListeners();
  }


```


Please, delete it from line 10


You don't need method `open` in this class because the parent class calls it